index.js

```diff
@@ -61,6 +61,17 @@ function magnetURIDecode (uri) {
     }
   })
 
+  if (result['x.pe']) {
+    // remove duplicates
+    if (Array.isArray(result['x.pe'])) {
```

Comment:
I meant that we should make `result['x.pe']` be always an array. This way we have a consistent output.

So something like this would do the trick:

```suggestion
    result['x.pe'] = Array.isArray(result['x.pe']) ? result['x.pe'] : [result['x.pe']]
```

Reply:
I've made the change 👍